index.hbs

```diff
@@ -22,6 +22,8 @@
                 {{/foreach}}
             </div>
 
+            {{pagination}}
```

Comment:
This template is for other pages than `/articles/` as well?

Reply:
This is actually a shorthand for calling the pagination template and filling it with data (if I get it right). Calling just the template itself (`{{> "pagination"}}`) shouldn't work.

https://ghost.org/docs/themes/helpers/pagination/

If you mean the `index.hbs` file, currently it's a template for all post listings: articles, categories, even newsletters…